components/backup/src/service.rs

```diff
@@ -50,25 +56,38 @@ where
         mut sink: ServerStreamingSink<CheckAdminResponse>,
     ) {
         let (tx, rx) = mpsc::unbounded();
-        self.router.broadcast_normal(|| {
-            PeerMsg::SignificantMsg(SignificantMsg::CheckPendingAdmin(tx.clone()))
-        });
-
-        let send_task = async move {
-            let mut s = rx.map(|resp| Ok((resp, WriteFlags::default())));
-            sink.send_all(&mut s).await?;
-            sink.close().await?;
-            Ok(())
-        }
-        .map(|res: Result<()>| match res {
-            Ok(_) => {
-                info!("check admin closed");
-            }
-            Err(e) => {
-                error!("check admin canceled"; "error" => ?e);
+        if let Some(router) = &self.router {
+            router.broadcast_normal(|| {
+                PeerMsg::SignificantMsg(SignificantMsg::CheckPendingAdmin(tx.clone()))
+            });
+            let send_task = async move {
+                let mut s = rx.map(|resp| Ok((resp, WriteFlags::default())));
+                sink.send_all(&mut s).await?;
+                sink.close().await?;
+                Ok(())
             }
-        });
-        ctx.spawn(send_task);
+            .map(|res: Result<()>| match res {
+                Ok(_) => {
+                    info!("check admin closed");
+                }
+                Err(e) => {
+                    error!("check admin canceled"; "error" => ?e);
+                }
+            });
+            ctx.spawn(send_task);
+        } else {
+            // check pending admin reqeust is used for EBS Backup.
+            // for raftstore v2. we don't need it for now. so just return unimplemented
+            ctx.spawn(
```

Comment:
Can we reuse [`unimplemented_call!`](https://docs.rs/grpcio/latest/grpcio/macro.unimplemented_call.html) here?

Reply:
Done, PTAL